src/print.c

```diff
@@ -25,27 +25,25 @@ static uint8_t mpix_gray8_to_256color(uint8_t gray8)
 void mpix_print_2_rows_truecolor(const uint8_t *top, const uint8_t *bot, size_t width)
 {
 	for (uint16_t w = 0; w < width; w++) {
-		mpix_port_printf("\e[48;2;%u;%u;%um\e[38;2;%u;%u;%um▄",
-			top[w * 3 + 0], top[w * 3 + 1], top[w * 3 + 2],
-			bot[w * 3 + 0], bot[w * 3 + 1], bot[w * 3 + 2]);
+		mpix_port_printf("\e[48;2;%u;%u;%um\e[38;2;%u;%u;%um▄", top[w * 3 + 0],
+				 top[w * 3 + 1], top[w * 3 + 2], bot[w * 3 + 0], bot[w * 3 + 1],
+				 bot[w * 3 + 2]);
 	}
 }
 
 void mpix_print_2_rows_256color(const uint8_t *top, const uint8_t *bot, size_t width)
 {
 	for (uint16_t w = 0; w < width; w++) {
-		mpix_port_printf("\e[48;5;%um\e[38;5;%um▄",
-			mpix_rgb24_to_256color(&top[w * 3]),
-			mpix_rgb24_to_256color(&bot[w * 3]));
+		mpix_port_printf("\e[48;5;%um\e[38;5;%um▄", mpix_rgb24_to_256color(&top[w * 3]),
+				 mpix_rgb24_to_256color(&bot[w * 3]));
 	}
 }
 
 void mpix_print_2_rows_256gray(const uint8_t *top, const uint8_t *bot, size_t width)
 {
 	for (uint16_t w = 0; w < width; w++) {
-		mpix_port_printf("\e[48;5;%um\e[38;5;%um▄",
-			mpix_gray8_to_256color(top[w]),
-			mpix_gray8_to_256color(bot[w]));
+		mpix_port_printf("\e[48;5;%um\e[38;5;%um▄", mpix_gray8_to_256color(top[w]),
+				 mpix_gray8_to_256color(bot[w]));
```

Comment:
It looks like clang-format did too many changes on this file, the indentation was a bit better before.